src/avfoundation.cs

```diff
@@ -3134,6 +3153,7 @@ interface AVCaptureSynchronizedDataCollection
 	interface IAVCaptureDataOutputSynchronizerDelegate {}
 	
 	[NoWatch, NoTV, iOS (11,0)]
+	[NoMac]
```

Comment:
that's not correct or it's incomplete as it produce
```
Removed Type MonoMac.AVFoundation.AVCaptureDataOutputSynchronizer

Removed Type MonoMac.AVFoundation.AVCaptureDataOutputSynchronizerDelegate

Removed Type MonoMac.AVFoundation.AVCaptureDataOutputSynchronizerDelegate_Extensions

Removed Type MonoMac.AVFoundation.IAVCaptureDataOutputSynchronizerDelegate

```
in our diffs

Reply:
part of `src/AVFoundation/AVCompat.cs` also needs to be back ported

Reply:
I back ported that in commit https://github.com/xamarin/xamarin-macios/pull/3605/commits/9eb9a4125ddae3f0233dbe969cb593844f2b2b85 and we still have the API diff issue:

https://jenkins.mono-project.com/job/xamarin-macios-pr-builder/6541/API_diff_(from_stable)/

Reply:
Ignore me, fixed in commit https://github.com/xamarin/xamarin-macios/pull/3605/commits/12de09ea9213535e9afaea354038be4233289f4e

Clearly not my week :/